dispatcher/backend/src/tests/integration/routes/business_logic/test_task_event_handler.py

```diff
@@ -0,0 +1,24 @@
+import db.models as dbm
+from common.enum import TaskStatus
+from common.utils import task_event_handler
+from db import Session
+
+
+class TestTaskEvents:
```

Comment:
I think both test names are not precise enough about what's being tested

Reply:
Fixed, is it better?
I find it's a bit long now, but I'm ok with it
FYI, I chose those vague names because I expected those tests to be enriched in the future, but it's true that it might never happen

Reply:
Yep